app/package.json

```diff
@@ -56,6 +56,7 @@
     "ai": "^3.2.35",
     "autoprefixer": "10.4.16",
     "bcrypt": "^5.1.1",
+    "chakra-react-select": "^4.9.1",
```

Comment:
We had a component like this before on the file upload form. Can you check if you can reuse the same thing as there instead of introducing this dependency?

Reply:
This has been changed in a follow up pr